1-exercises/B-object-arrays/exercise-2.js

```diff
@@ -15,11 +15,13 @@ let store = [
 ];
 
 let money = 1.5;
+let check = 0;
 
-let thingsICanBuy =
-  /* todo */
+let thingsICanBuy = store
```

Comment:
Great. Just one thing that can be consider is how to choose the variable name. like item. if it is a little bit meaningful then your code is more readable